addons/account_payment/views/account_portal_templates.xml

```diff
@@ -16,23 +14,41 @@
         <xpath expr="//t[@t-foreach='invoices']/tr/td[hasclass('tx_status')]" position="replace">
             <t t-set="last_tx" t-value="invoice.get_portal_last_transaction()"/>
             <td class="tx_status text-center">
-                <t t-if="invoice.state == 'posted' and invoice.payment_state in ('not_paid', 'partial') and (last_tx.state not in ['pending', 'authorized', 'done', 'cancel'] or (last_tx.state == 'pending' and last_tx.provider_code in ('none', 'custom')))">
-                    <span class="badge rounded-pill text-bg-info"><i class="fa fa-fw fa-clock-o"></i><span class="d-none d-md-inline"> Waiting for Payment</span></span>
+                <t t-if="last_tx">
+                    <!-- c/p of account_payment -->
```

Comment:
You can drop the comment now.

Reply:
dropped by second commit (should i merge them ?)

Reply:
No, it's clearer with two commits. If you feel like it you can amend the first commit :p